### PR TITLE
chore: deps update

### PR DESCRIPTION
**Pull request by alvarosabu**

### CODE REVIEW

File: lib/tsconfig.json [hunk 1]
```diff
@@ -5,7 +5,8 @@
     "strict": false,
     "declaration": true,
     "declarationDir": "dist",
-    "emitDeclarationOnly": true
+    "emitDeclarationOnly": true,
+    "ignoreDeprecations": "5.0"
```
Comment (Dawntraoz) [inline lib/tsconfig.json:9]: Oh, so this was the fix to the broken CI? I'm curious to know what happened. Good catch <3

**Review by Dawntraoz — APPROVED** [review]

**Review by Dawntraoz — CHANGES_REQUESTED** [review]
I see what you did there 😏, but would it be better to fix the issue before 5.5 comes in and stop working?

> Option 'importsNotUsedAsValues' is deprecated and will stop functioning in TypeScript 5.5. Specify compilerOption '"ignoreDeprecations": "5.0"' to silence this error.
  Use 'verbatimModuleSyntax' instead.

**Comment by alvarosabu** [comment]
> importsNotUsedAsValues

That would have been my preferred option, but `importsNotUsedAsValues` is not even set on non of the tsconfigs of our repository (if you try to search for it on the repo you will not find it)

**Comment by alvarosabu** [comment]
@alexjoverm @Dawntraoz seems like the warning comes from this line on the tsconfig

```
 "extends": "@vue/tsconfig/tsconfig.json"
```
Screenshot 2023-04-03 at 08 54 51

Seems like `@vue` package hasn't replace `importsNotUsedAsValues` and `preserveValueImports`  with `verbatimModuleSyntax` yet, that's why the error.

I see 2 options:

1.  Continue with the suggested action from the error until vue fixes it from their end.
```
error TS5101: Option 'importsNotUsedAsValues' is deprecated and will stop functioning in TypeScript 5.5. Specify compilerOption '"ignoreDeprecations": "5.0"' to silence this error.
  Use 'verbatimModuleSyntax' instead.
  ```
  
 or 
 
 2. Drop the extending of `@vue/tsconfig/tsconfig.json` for our own tsconfig.

wdyt?

**Pull request closed without merging**